Lib/glyphsLib/classes.py

```diff
@@ -1234,8 +1239,20 @@ def shouldWriteValueForKey(self, key):
         if key in ("xHeight", "capHeight", "ascender"):
             # Always write those values
             return True
+        if key == "name":
+            if self.font and int(self.font.appVersion) >= 1075:
+                return True
+            return False
```

Comment:
I don't think we need to check the appVersion of the original file to decide whether to write the "name" attribute.
If the attribute is not None (because the file comes from new Glyphs), then we write it, otherwise we need to do what Glyphs.app does when opening and re-saving a file generated from an older version of the app. I don't know the answer to that (don't a mac at hand to try this out ATM).
Basically, we should "upgrade" the source files to whatever latest version of Glyphs.app that we currently match at any given time. Instead of attempting to ensure we roundtrip any file generated from previous versions. I think this only adds more complexity.

Reply:
I’m all for keeping it simple and just supporting the latest version.